x-pack/plugins/search_inference_endpoints/public/components/all_inference_endpoints/render_table_columns/render_endpoint/endpoint_info.tsx

```diff
@@ -154,6 +156,23 @@ function mistralAttributes(endpoint: InferenceAPIConfigResponse) {
     .join(', ');
 }
 
+function amazonBedrockAttributes(endpoint: InferenceAPIConfigResponse) {
+  const serviceSettings = endpoint.service_settings;
+
+  const region = 'region' in serviceSettings ? serviceSettings.region : undefined;
+  const provider =
+    'provider' in serviceSettings ? serviceSettings.provider.toLocaleLowerCase() : undefined;
+  const model = 'model' in serviceSettings ? serviceSettings.model : undefined;
+
+  return [
+    region && `region: ${region}`,
+    provider && `provider: ${provider}`,
+    model && `model: ${model}`,
```

Comment:
We do not need to add 'model' information to this array. This should be automatically handled by this [code-block](https://github.com/elastic/kibana/blob/cbb91f1f6beb457a59710ed8d159d56e20d42aed/x-pack/plugins/search_inference_endpoints/public/components/all_inference_endpoints/render_table_columns/render_endpoint/endpoint_info.tsx#L34-L47). Essentially, we display models using a badge for different inference endpoints.